elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/Context.java

```diff
@@ -102,17 +103,14 @@ private Object get(Object key, Map<String, ? extends Object> fixedArgs) {
         if (this.queryable.hasJoin(keyStr)) {
             SQLJoin sqlJoin = this.queryable.getJoin(keyStr);
             Queryable joinQueryable = metaDataStore.getTable(sqlJoin.getJoinTableType());
-            TableContext newCtx = TableContext.builder()
+            Context joinCtx = Context.builder()
                             .queryable(joinQueryable)
-                            .alias(appendAlias(alias, keyStr))
-                            .metaDataStore(metaDataStore)
+                            .alias(appendAlias(this.alias, keyStr))
+                            .metaDataStore(this.metaDataStore)
+                            .queriedColArgs(this.queriedColArgs)
                             .build();
 
-            // Copy $$column to join context.
-            if (this.containsKey(COL_PREFIX)) {
-                newCtx.put(COL_PREFIX, this.get(COL_PREFIX));
-            }
-            return newCtx;
+            return joinCtx;
```

Comment:
table arguments are not required in join context as it will use default table arguments for join table, column arguments are propogated via queriedColArgs property in join context